src/app/views/dashboard/games/manage/manage.html

```diff
@@ -96,28 +101,50 @@ <h1 class="section-header">
 						<translate>Devlog</translate>
 					</router-link>
 				</li>
-				<li v-app-tooltip.bottom="$gettext( `dash.games.api_tooltip` )">
-					<router-link
-						:to="{ name: 'dash.games.manage.api.overview' }"
-						:class="{ active: $route.name.indexOf( 'dash.games.manage.api' ) === 0 }"
-						>
-						<translate>dash.games.api_tab</translate>
-					</router-link>
-				</li>
-				<li v-app-tooltip.bottom="$gettext( `Manage your game keys and give access to users.` )">
-					<router-link
-						:to="{ name: 'dash.games.manage.key-groups.list' }"
-						:class="{ active: $route.name.indexOf( 'dash.games.manage.key-groups' ) === 0 }"
-						>
-						<translate>Keys/Access</translate>
-					</router-link>
-				</li>
-				<li v-app-tooltip.bottom="$gettext( `Game Jolt Sites are customizable external sites for your portfolio and games!` )">
+				<app-game-perms required="game-api">
```

Comment:
this needs to be tag="li". you can't have any other tags besides <li> within <ul> according to the standard